src/MaplibreGLLayer.js

```diff
@@ -184,8 +184,10 @@ export var MaplibreGLJSLayer = Layer.extend({
     );
 
     // allow GL base map to pan beyond min/max latitudes
-    this._glMap.transform.latRange = null;
-    this._glMap.transform.maxValidLatitude = Infinity;
+    // TODO: PR NOTE: THIS DOES NOT WORK WITH MAPLIBRE GL v5 SO THIS ISSUE IS BACK
+    // ... NEED TO RESOLVE BEFORE MERGING PR?
+    // this._glMap.transform.latRange = null;
+    // this._glMap.transform.maxValidLatitude = Infinity;
```

Comment:
@gavinr-maps in https://github.com/Esri/esri-leaflet-vector/issues/130#issuecomment-1215740071 I talked about an alternate fix for this with `maxBounds` does that still work?

Would setting `maxBounds`, `worldCopyJump` to `false` and `maxBoundsViscosity` to `1.0` would resolve this?